doc/nvim-ide.txt

```diff
@@ -534,34 +534,55 @@ Config:
         default_height = nil,
         -- disable all keymaps for the Explorer component.
         disabled_keymaps = false,
-        keymaps = {
-            expand = "zo",
-            collapse = "zc",
-            collapse_all = "zM",
-            edit = "<CR>",
-            edit_split = "s",
-            edit_vsplit = "v",
-            edit_tab = "t",
-            hide = "<C-[>",
-            close = "X",
-            new_file = "n",
-            delete_file = "D",
-            new_dir = "d",
-            rename_file = "r",
-            move_file = "m",
-            copy_file = "p",
-            select_file = "<Space>",
-            deselect_file = "<Space><Space>",
-            change_dir = "cd",
-            up_dir = "..",
-            file_details = "i",
-            toggle_exec_perm = "*",
-            maximize = "+",
-            minimize = "-"
-        },
+        keymaps = require('ide.explorer.components.presets').default
     }
 <
 
+The following keymapping presets are available in the
+`require('ide.explorer.components.presets')` module:
```

Comment:
So, I'm just going back and forth on whether documenting the table merging here is good for user facing docs. 

We are basically asking the reader to merge the table in their heads, which isnt too bad, but also would be nicer to just print the full table and keep how we merge it as an internal detail?

Reply:
Yeah I can print the table and just put the full table in the docs 👍

Reply:
sure, if you agree with that idea. It may just be a bit nicer for the reader.

Reply:
done